public/app/features/alerting/unified/utils/config.ts

```diff
@@ -6,7 +6,7 @@ export function getAllDataSources(): Array<DataSourceInstanceSettings<DataSource
 }
 
 export function checkEvaluationIntervalGlobalLimit(alertGroupEvaluateEvery?: string) {
-  if (!alertGroupEvaluateEvery) {
+  if (!alertGroupEvaluateEvery || !isValidGoDuration(alertGroupEvaluateEvery)) {
```

Comment:
This also fixes a bug when you input `)` or something else that is nonsensical in the alert creation flow (or anywhere else where we call `checkEvaluationIntervalGlobalLimit`)

Reply:
❤️